src/Core/Geographic/Extent.js

```diff
@@ -194,7 +194,12 @@ Extent.prototype.offsetToParent = function offsetToParent(other) {
     const scale =
```

Comment:
you could standardize variable names in the caller `scaleX` instead of `scale`

Reply:
Fixed.